modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -298,41 +299,43 @@ spec:
   configuration:
     velero:
       defaultPlugins:
-        - kubevirt # <2>
-        - gcp # <3>
-        - csi # <4>
-        - openshift # <5>
-      resourceTimeout: 10m # <6>
-    nodeAgent: # <7>
-      enable: true # <8>
-      uploaderType: kopia # <9>
+        - aws # <2>
```

Comment:
Attach BSL spec has only one backupLocation. Only one plugin is required either AWS or GCP

Reply:
@PrasadJoshi12 - updated. Thanks